ols/app/models/config.py

```diff
@@ -271,27 +271,62 @@ def validate(self) -> None:
         pass
 
 
-class LoggerConfig(BaseModel):
-    """Logger configuration."""
+class LoggingConfig(BaseModel):
+    """Logging configuration."""
 
-    default_level: Optional[int | str] = None
+    app_log_level: Optional[str] = None
+    httpcore_log_level: Optional[str] = None
+    httpx_log_level: Optional[str] = None
+    urllib3_log_level: Optional[str] = None
 
     def __init__(self, data: Optional[dict] = None):
         """Initialize configuration and perform basic validation."""
         super().__init__()
         if data is None:
             return
-        level = logging.getLevelName(data.get("default_level", "INFO"))
-        if level is None:
+        app_log_level = logging.getLevelName(data.get("app_log_level", "info").upper())
+        if app_log_level is None:
             raise InvalidConfigurationError(
-                f"invalid log level for default log: {data.get('default_level',None)}"
+                f"invalid log level for default log: {data.get('app_log_level')}"
             )
-        self.default_level = level
+        self.app_log_level = app_log_level
+
+        httpcore_log_level = logging.getLevelName(
+            data.get("httpcore_log_level", "warning").upper()
+        )
+        if httpcore_log_level is None:
+            raise InvalidConfigurationError(
+                f"invalid log level for default log: {data.get('httpcore_log_level')}"
+            )
+        self.httpcore_log_level = httpcore_log_level
+
+        httpx_log_level = logging.getLevelName(
+            data.get("httpx_log_level", "warning").upper()
+        )
+        if httpx_log_level is None:
+            raise InvalidConfigurationError(
+                f"invalid log level for default log: {data.get('httpx_log_level')}"
+            )
+        self.httpx_log_level = httpx_log_level
+
+        urllib3_log_level = logging.getLevelName(
+            data.get("urllib3_log_level", "warning").upper()
+        )
+        if urllib3_log_level is None:
+            raise InvalidConfigurationError(
+                f"invalid log level for default log: {data.get('urllib3_log_level')}"
+            )
+        self.urllib3_log_level = urllib3_log_level
 
     def __eq__(self, other):
         """Compare two objects for equality."""
-        if isinstance(other, LoggerConfig):
-            return self.default_level == other.default_level
+        if isinstance(other, LoggingConfig):
+            return (
+                self.app_log_level == other.app_log_level
+                and self.httpcore_log_level == other.httpcore_log_level
+                and self.httpx_log_level == other.httpx_log_level
+                and self.urllib3_log_level == other.urllib3_log_level
+            )
         return False
 
     def validate(self) -> None:
```

Comment:
@onmete We've renamed the validate() -> validate_yaml() or it clashed with a base class method, in https://github.com/openshift/lightspeed-service/pull/240. Just a heads up.